mx.py

```diff
@@ -1713,15 +1716,21 @@ def _getCompiler(self):
         if self.args.jdt and not self.args.force_javac:
             return ECJCompiler(self.args.jdt, self.args.extra_javac_args)
         else:
-            return JavacCompiler(self.args.alt_javac, self.args.extra_javac_args)
+            if self.args.no_javac_daemon:
+                return JavacCompiler(self.args.alt_javac, self.args.extra_javac_args)
+            else:
+                return JavacCompilerDaemon(self.args.alt_javac, self.args.extra_javac_args)
```

Comment:
The comment isn’t visible in the PR for some reason.  Removing it would simplify things.  What alternate javac is there besides ecj?

> On Mar 11, 2016, at 1:04 AM, Gilles Duboscq <notifications@github.com> wrote:
> 
> In mx.py <https://github.com/graalvm/mx/pull/72#discussion_r55804902>:
> 
> > @@ -1713,15 +1716,21 @@ def _getCompiler(self):
> >          if self.args.jdt and not self.args.force_javac:
> >              return ECJCompiler(self.args.jdt, self.args.extra_javac_args)
> >          else:
> > -            return JavacCompiler(self.args.alt_javac, self.args.extra_javac_args)
> > +            if self.args.no_javac_daemon:
> > +                return JavacCompiler(self.args.alt_javac, self.args.extra_javac_args)
> > +            else:
> > +                return JavacCompilerDaemon(self.args.alt_javac, self.args.extra_javac_args)
> It's about using a different javac than the one from the JDK.
> I don't think it's used anywhere. Maybe we should just remove the support.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/graalvm/mx/pull/72/files#r55804902>.
>